--- a/Cartographer/Libs/Babble-Spell-2.2/Babble-Spell-2.2.lua	2015-03-05 15:28:56.196305931 +0100
+++ b/Cartographer/Libs/Babble-Spell-2.2/Babble-Spell-2.2.lua	2015-03-05 15:28:56.020305924 +0100
@@ -25,7 +25,7 @@ local BabbleSpell = AceLibrary("AceLocal
 -- uncomment below for debug information
 -- BabbleSpell:EnableDebugging()
 
-BabbleSpell:RegisterTranslations("enUS", function()
+--[[ BabbleSpell:RegisterTranslations("enUS", function()
 	return {
 		["Abolish Disease"] = true,
 		["Abolish Poison Effect"] = true,
@@ -1061,7 +1061,7 @@ BabbleSpell:RegisterTranslations("enUS",
 		["Wrath"] = true,
 		["Wyvern Sting"] = true,
 	}
-end)
+end) ]]
 
 BabbleSpell:RegisterTranslations("deDE", function()
 	return {
--- a/Cartographer/Libs/Babble-Zone-2.2/Babble-Zone-2.2.lua	2015-03-21 12:25:08.031071311 +0100
+++ b/Cartographer/Libs/Babble-Zone-2.2/Babble-Zone-2.2.lua	2015-03-05 15:28:56.020305924 +0100
@@ -26,7 +26,7 @@ local BabbleZone = AceLibrary("AceLocale
 -- uncomment below for debug information
 -- BabbleZone:EnableDebugging()
 
-BabbleZone:RegisterTranslations("enUS", function()
+--[[ BabbleZone:RegisterTranslations("enUS", function()
 	return {
 		["Ahn'Qiraj"] = true,
 		["Alterac Mountains"] = true,
@@ -172,7 +172,7 @@ BabbleZone:RegisterTranslations("enUS",
 		["Caverns of Time"] = true,
 		["Zul'Aman"] = true,
 	}
-end)
+end) ]]--
 
 BabbleZone:RegisterTranslations("deDE", function()
 	return {
@@ -190,8 +190,8 @@ BabbleZone:RegisterTranslations("deDE",
 		["Blackrock Depths"] = "Blackrocktiefen",
 		["Blackrock Mountain"] = "Der Blackrock",
 		["Blackrock Spire"] = "Blackrockspitze",
-    ["Lower Blackrock Spire"] = true,
-    ["Upper Blackrock Spire"] = true,
+		["Lower Blackrock Spire"] = "Untere Schwarzfelsspitze",
+		["Upper Blackrock Spire"] = "Obere Schwarzfelsspitze",
 		["Blackwing Lair"] = "Pechschwingenhort",
 		["Blasted Lands"] = "Verw\195\188stete Lande",
 		["Booty Bay"] = "Booty Bay",
@@ -203,6 +203,9 @@ BabbleZone:RegisterTranslations("deDE",
 		["Deeprun Tram"] = "Die Tiefenbahn",
 		["Desolace"] = "Desolace",
 		["Dire Maul"] = "D\195\188sterbruch",
+		["Dire Maul (East)"] = "D\195\188sterbruch (Ost)",
+		["Dire Maul (West)"] = "D\195\188sterbruch (West)",
+		["Dire Maul (North)"] = "D\195\188sterbruch (Nord)",
 		["Dun Morogh"] = "Dun Morogh",
 		["Durotar"] = "Durotar",
 		["Duskwood"] = "D\195\164mmerwald",
